packages/config/config/devices/0x031e/lzw36.json

```diff
@@ -0,0 +1,559 @@
+            "minValue": 0,
+            "maxValue": 10,
+            "defaultValue": 5,
+            "unsigned": true,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "22": {
+            "label": "Light LED Strip Intensity (When OFF)",
+            "description": "This is the intensity of the Light LED strip when the switch is off. This is useful for users to see the light switch location when the lights are off.\n      Range: 0-10\n      Default: 1",
+            "valueSize": 1,
+            "minValue": 0,
+            "maxValue": 10,
+            "defaultValue": 1,
+            "unsigned": true,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "23": {
+            "label": "Fan LED Strip Intensity (When OFF)",
+            "description": "This is the intensity of the Fan LED strip when the switch is off. This is useful for users to see the light switch location when the lights are off.\n      Range: 0-10\n      Default: 1",
+            "valueSize": 1,
+            "minValue": 0,
+            "maxValue": 10,
+            "defaultValue": 1,
+            "unsigned": true,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "24": {
+            "label": "Light LED Strip Effect",
+            "description": "Please see website for documentation.\n          Byte 4: Effect - 0 = Off, 1 = Solid, 2 = Slow Blink, 3 = Fast Blink, 4 = Chase, 5 = Pulse\n          Byte 3: Duration - 1 to 60 = seconds, 61 to 120  minutes, 121 -  254 = hours, 255 = Indefinitely\n          Byte 2: Intensity - 0 to 9.  0 = dim, 9 = bright\n          Byte 1: Color - 0 - 255. Hue color wheel. value/255 * 360 = Hue color wheel value\n      Range: 0-100600319\n      Default: 0",
+            "valueSize": 4,
+            "minValue": 0,
+            "maxValue": 100600319,
+            "defaultValue": 0,
+            "unsigned": true,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "25": {
+            "label": "Fan LED Strip Effect",
+            "description": "Please see website for documentation.\n          Byte 4: Effect - 0 = Off, 1 = Solid, 2 = Slow Blink, 3 = Fast Blink, 4 = Chase, 5 = Pulse\n          Byte 3: Duration - 1 to 60 = seconds, 61 to 120  minutes, 121 -  254 = hours, 255 = Indefinitely\n          Byte 2: Intensity - 0 to 9.  0 = dim, 9 = bright\n          Byte 1: Color - 0 - 255. Hue color wheel. value/255 * 360 = Hue color wheel value\n      Range: 0-100600319\n      Default: 0",
+            "valueSize": 4,
+            "minValue": 0,
+            "maxValue": 100600319,
+            "defaultValue": 0,
+            "unsigned": true,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
```

Comment:
Looks like these two should be split into [partial parameters](https://zwave-js.github.io/node-zwave-js/#/development/config-files?id=partial-parameters)